CONTRIBUTING.md

```diff
@@ -1,3 +1,533 @@
 # Contribution Guidelines
 
 We follow the [Contribution Guidelines of PySyft](https://github.com/OpenMined/PySyft/blob/dev/CONTRIBUTING.md), please refer to them to start developing!
+
+
+- [Contribution Guidelines](#contribution-guidelines)
+  - [Issues / PRs](#issues--prs)
+    - [Beginner Issues](#beginner-issues)
+  - [Requirements](#requirements)
+    - [Linux](#linux)
+    - [MacOS](#macos)
```

Comment:
Q: Could you check if `sycret` ahs support for Windows and MacOS?

Reply:
I checked https://github.com/OpenMined/sycret and it does not mention it. Rust can be installed in all systems so I suppose there should not be any problem (source: https://forge.rust-lang.org/infra/other-installation-methods.html). Maybe @tholop can give us a definitive answer.

Also, before going to master @gmuraru. Should I remove all the macos and windows parts that you told me to in the reviews? Should I remove the initial part where we say that we follow PySyft contributions?

Reply:
No no, if `torchcsprang` is also available there, I think we are good for the moment.

Reply:
@danielorihuela @gmuraru I tested Sycret on Linux and MacOS. It should work on Windows but I never tried. 
How are you guys testing SyMPC on Windows? Do you have a special workflow for that (I couldn't find it) or do you spam someone who is running Windows?

Reply:
We do not have actions for Windows and OSX. @danielorihuela could you add ```Windows``` and ```MacOS``` in the `.github` workflow, for PR-tests and lets see if they work

Reply:
Okay, I am gonna give it a try.

Reply:
Thanks @danielorihuela! If it works I'll do the same for Sycret :slightly_smiling_face:

Reply:
I think you only need to change this to have multiple values? `runs-on: ubuntu-latest` (the PySyft team is doing this)

Reply:
Yeah I saw, thanks!

I made it work yesterday with MacOS. The problem is with windows. It does not have any command "hash -r", so I removed it. But then the trilom github action fails. I do not really know if it is related or not.

![image](https://user-images.githubusercontent.com/20444151/118774393-14dba700-b886-11eb-95aa-19a2d4c97533.png)

Reply:
Could you also rebase, I made a change to one of the workflow files to have a new version for `trilom`

Reply:
Yeah, I tried that yesterday too. 
![image](https://user-images.githubusercontent.com/20444151/118777773-b9131d00-b889-11eb-8ee8-979d91f85988.png)

Reply:
Maybe this issue has something to do https://github.com/trilom/file-changes-action/issues/112

Reply:
@gmuraru I could make the tests works by changing the github action to get all the changed files (jitterbit/get-changed-files@v1). You can see a run https://github.com/danielorihuela/SyMPC/pull/12/checks?check_run_id=2632889341.

Reply:
Yep! It looks good with the change from there